internal/backends/winit/winitwindowadapter.rs

```diff
@@ -685,6 +685,28 @@ impl WinitWindowAdapter {
             })
             .ok()
     }
+
+    pub fn activation_changed(&self, is_active: bool) {
+        let have_focus = is_active || self.input_method_focused();
+        let slint_window = self.window();
+        let runtime_window = WindowInner::from_pub(slint_window);
+        // We don't render popups as separate windows yet, so treat
+        // focus to be the same as being active.
+        if have_focus != runtime_window.active() {
+            slint_window
+                .dispatch_event(corelib::platform::WindowEvent::WindowActiveChanged(have_focus));
+        }
+
+        #[cfg(all(feature = "muda", target_os = "macos"))]
+        {
+            if self.muda_adapter.borrow().is_none() {
+                *self.muda_adapter.borrow_mut() =
+                    Some(crate::muda::MudaAdapter::setup_default_menu());
+            }
```

Comment:
I believe that's
```suggestion
            self.muda_adapter.borrow_mut().get_or_insert_with(crate::muda::MudaAdapter::setup_default_menu);
            
```

Reply:
Ok :)

Reply:
Slight correction: setup_default_menu can fail.